logging/cluster-logging-release-notes.adoc

```diff
@@ -8,34 +8,65 @@ toc::[]
 [id="openshift-logging-about-this-release"]
 == About this release
 
-The following advisories are available for Red Hat OpenShift Logging:
+The following advisories are available for Red Hat OpenShift Logging versions 5.1.x and 5.2.x:
 
+* link:https://access.redhat.com/errata/RHBA-2021:2885[RHBA-2021:2885 - Bug Fix Advisory. Openshift Logging Bug Fix Release 5.1.1]
 * link:https://access.redhat.com/errata/RHBA-2021:2112[RHBA-2021:2112 - Bug Fix Advisory. OpenShift Logging Bug Fix Release 5.1.0]
-* link:https://access.redhat.com/errata/RHBA-2021:2655[RHBA-2021:2655 - Bug Fix Advisory. Openshift Logging Bug Fix Release (5.0.6)]
-* link:https://access.redhat.com/errata/RHSA-2021:2374[RHSA-2021:2374 - Security Advisory. Moderate: Openshift Logging Bug Fix Release (5.0.5)]
-* link:https://access.redhat.com/errata/RHSA-2021:2136[RHSA-2021:2136 - Security Advisory. Moderate: Openshift Logging security and bugs update (5.0.4)]
-* link:https://access.redhat.com/errata/RHSA-2021:1515[RHSA-2021:1515 - Security Advisory. Important OpenShift Logging Bug Fix Release (5.0.3)]
-* link:https://access.redhat.com/errata/RHBA-2021:1167[RHBA-2021:1167 - Bug Fix Advisory. OpenShift Logging Bug Fix Release (5.0.2)]
-* link:https://access.redhat.com/errata/RHBA-2021:0963[RHBA-2021:0963 - Bug Fix Advisory. OpenShift Logging Bug Fix Release (5.0.1)]
-* link:https://access.redhat.com/errata/RHBA-2021:0652[RHBA-2021:0652 - Bug Fix Advisory. Errata Advisory for Openshift Logging 5.0.0]
 
 [id="openshift-logging-supported-versions"]
 == Supported versions
 
-* OpenShift Logging version 5.0 runs on {product-title} versions 4.7 and 4.8.
-* OpenShift Logging version 5.1 runs on {product-title} versions 4.7 and 4.8.
+* OpenShift Logging versions 5.1 and 5.2 run on {product-title} versions 4.7 and 4.8.
 
 [id="openshift-logging-inclusive-language"]
 == Making open source more inclusive
 
 Red Hat is committed to replacing problematic language in our code, documentation, and web properties. We are beginning with these four terms: master, slave, blacklist, and whitelist. Because of the enormity of this endeavor, these changes will be implemented gradually over several upcoming releases. For more details, see link:https://www.redhat.com/en/blog/making-open-source-more-inclusive-eradicating-problematic-language[Red Hat CTO Chris Wright’s message].
 
 // Release Notes by version
+
+[id="cluster-logging-release-notes-5-2-0"]
+== OpenShift Logging 5.2.0
```

Comment:
I think this could be it's own module as per https://github.com/openshift/openshift-docs/pull/35445/files#diff-39d0bf3cca6d5b60bc65b58d2fe0b7c791cefa2b64f8e915729dc4960b88162bR72 ?

Reply:
Unfortunately, because OCP still forbids xrefs in modules, I'm bringing this module content into the assembly file. This way, users can follow the xref links from the release note to the relevant documentation.

Reply:
Makes sense! Thanks